### PR TITLE
Efk logging pbench

### PR DESCRIPTION
**Pull request by rflorenc**
This is ready to use but should be considered WIP.

### CODE REVIEW

File: logging_metrics_performance/enterprise_logging/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+E2Es:
+
+Relevant Until https://github.com/kubernetes/kubernetes/pull/24536/files merges into upstream kube and openshift...
+
+To run the e2es, you essentially follow these steps.
+
+- git clone jayunit100/kubernetes
+- cd kubernetes and checkout branch "LoggingSoak"
```
Comment (timothysc) [inline logging_metrics_performance/enterprise_logging/README.md]: Could you mark as a TODO to change instructions once this lands in a rebase to just run from extended.

**Comment by timothysc** [comment]
I think for now I'm ok with a lgtm to get this in and done, but we're probably going to need to refine.

File: logging_metrics_performance/enterprise_logging/README.md (hunk 1 above)
Comment (rflorenc) [inline logging_metrics_performance/enterprise_logging/README.md]: Sure.

**Pull request merged**